grr-ui/Test/testSampleTest.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('chai').assert;
+const app = require('../src/js/testSample');
+
+//Results
+sayHelloResult = app.sayHello();
+addNumbersResult = app.addNumbers(5,5);
+
+describe('App', function(){
```

Comment:
How are you going to test the actual functions we have? The sample stuff looks nice but it'd be better if we at least had a test example for one of our current functions.